crossing-training.lic

```diff
@@ -315,6 +317,21 @@ class CrossingTraining
       fput('cast')
     end
 
+    if MOON_SPELLS.include? data['abbrev']
+      moon = UserVars.moons['visible'].first
+      unless moon
+        echo "No moon available to cast #{data['abbrev']}"
+        unless Script.running? 'moonwatch'
+          echo "moonwatch is not running. Starting it now"
+          custom_require 'moonwatch'
+          echo "Run `;autostart add moonwatch` to avoid this in the future"
```

Comment:
dependency manager manages our autostarts, nothing of ours should be in ;autostart other than dependency. use ;e autostart('scriptname')